content/influxdb/v2.0/reference/cli/influxd/upgrade.md

```diff
@@ -49,6 +49,7 @@ influxd upgrade [command]
 |      | `--log-level`                   | Supported log levels are `debug`, `info`, `warn`, and `error` (default `info`)                             | string     |
```

Comment:
Noticed that the `---influx-configs-path` option is missing. The text from `influxd upgrade -h` is:
```
-c, --influx-configs-path string            path for 2.x CLI configurations file (default "~/.influxdbv2/configs")
```